arangod/IResearch/IResearchRocksDBRecoveryHelper.h

```diff
@@ -61,6 +62,10 @@ class IResearchRocksDBRecoveryHelper final : public RocksDBRecoveryHelper {
 
   virtual void prepare() override;
 
+  void unprepare() noexcept  override { 
+    _cookies.clear();
```

Comment:
```suggestion
    _cookies = {};
```
_cookies used only once?

Reply:
Yes, We discard all "pre-recovery" snapshots once recovery is finished